tests/quantization/test_quantized_ops.py

```diff
@@ -409,16 +410,19 @@ def test_all_arith_ops(
     check_r2_score(raw_output_cv, quantized_output_cv)
 
     # Check that we get the same fp32 results in V+V (if supported), V+C and C+V modes
-    if supports_enc_with_enc:
+    if supports_enc_with_enc and n_bits > 16:
```

Comment:
same

Reply:
We were checking correctness for low bit-width. Worked fine for univariate but now QuantizedMul|Div have a quant requant step which impact the correctness.

Reply:
I answered again above (https://github.com/zama-ai/concrete-ml/pull/690/files#r1677535203)

Reply:
updated to have tight n_bits threshold for each qops